qutebrowser/completion/models/base.py

```diff
@@ -44,6 +44,17 @@ class BaseCompletionModel(QStandardItemModel):
         COLUMN_WIDTHS: The width percentages of the columns used in the
                         completion view.
         DUMB_SORT: the dumb sorting used by the model
+
+    Attributes:
+        filtered_out_cache:
```

Comment:
I actually tried that first, but the problem is that the completer creates a new `sortfilter.CompletionFilterModel` instance on every `update_completion` call, so the cache would not persist there...